Bump workflow version

michaeldiscala opened the pull request:
This patch just relaxes the dependency on workflow version 1.1. I've tested workflow-sequel manually with the newer version of workflow and everything seems to be functioning as expected.

tmaier commented:
Thank you.

I am on vacation, but I want to give you quick, initial feedback:
- I prefer at least some minor constraint, like < 2.0
- please add an entry to the changelog

I will be back from vacation in exactly one week. So if I forgot to take care of your pull request within 2 weeks, please remind me again

Best

Tobias

michaeldiscala commented:
Hi Tobias,

Thanks for your feedback! I've just added another commit addressing those points. Please let me know if you'd prefer a different format for the changelog.

I hope you're enjoying your vacation!

--Mike

tmaier commented:
Hi Mike,

thank you for your contribution.
I merged your changes and released a new version.
Version 0.1.0
